spec/providers/firestore.spec.ts

```diff
@@ -252,19 +254,19 @@ describe('Firestore Functions', () => {
     }).timeout(5000);
 
     it('constructs appropriate fields and getters for event.data on "document.create" events', () => {
-      let testFunction = firestore
+      const testFunction = firestore
         .document('path')
         .onCreate((data, context) => {
           expect(data.data()).to.deep.equal({ key1: true, key2: 123 });
           expect(data.get('key1')).to.equal(true);
           return true; // otherwise will get warning about returning undefined
         });
-      let data = constructEvent({}, createValue(), 'document.create');
-      return testFunction(data.data, data.context);
+      const event = constructEvent({}, createValue(), 'document.create');
```

Comment:
if we change this to event should we change the other ones as well?

Reply:
Changing it back to data here as well.

Reply:
Actually, this makes sense to be event, so changed the others to event instead of data.